ner_v2/api.py

```diff
@@ -72,6 +72,7 @@ def parse_post_request(request):
         PARAMETER_TIMEZONE: request_data.get('timezone'),
         PARAMETER_LANGUAGE_SCRIPT: request_data.get('language_script', ENGLISH_LANG),
         PARAMETER_SOURCE_LANGUAGE: request_data.get('source_language', ENGLISH_LANG),
+        PARAMETER_PAST_DATE_REFERENCED: request_data.get('date_past_reference', 'False'),
```

Comment:
`False` in string? is this correct?

Reply:
Yes, this is correct. We are handling it below.